src/js/utils/element-utils.js

```diff
@@ -59,11 +63,69 @@ export function getOrInsertCSSRule(styleParent, selectorText) {
     return {
       style: {
         setProperty: () => {},
-        removeProperty: () => {}
-      }
+        removeProperty: () => {},
+      },
     };
   }
 
   style.sheet.insertRule(`${selectorText}{}`, style.sheet.cssRules.length);
   return style.sheet.cssRules[style.sheet.cssRules.length - 1];
 }
+
+/**
+ * Gets the number represented by the attribute
+ * @param {HTMLElement} el
+ * @param {string} attrName
+ * @returns {number | undefined} Will return undefined if no attribute set
+ */
+export function getNumericAttr(el, attrName) {
+  const attrVal = el.getAttribute(attrName);
+  return attrVal != null ? +attrVal : undefined;
```

Comment:
Undefined feels like the wrong value to return here. We're calling this a number type so I think `NaN` could/should be the falsey value, like the duration prop on the video el.

(I also think the `mediaDuration` prop where we currently return `undefined` is wrong)

Reply:
I'm torn on this one. I agree for sure it's easier to work with `undefined` and it would be handy to use this util as 

`getNumericAttr(...) ?? 0` or `getNumericAttr(...) ?? NaN`

which we do in Mux elements.

I'm leaning to also return `undefined` for the Element property because of the reflection of attr / prop.
it seems easier they would be more interchangeable for checking nullish  `el.mediaDuration == null` and `el.getAttribute('mediaduration') == null`